### ISSUE TITLE
Sharing jail followup

### ISSUE THREAD

**Issue by kulmann** [owncloud/web#6899]
https://github.com/owncloud/web/pull/6593 has been merged - there are some followups necessary:

* Spaces:
    * [x] Share a pdf-file inside a space, then go to "Shared with others page" -> this file can't be opened in the PDF-viewer (probably wrong request URL: `remote.php/dav/files/moss/sample3.txt`) Fixed via https://github.com/owncloud/web/pull/7379
    * [x] #7382
    *  [x] Uploading a file to a space root -> the parent folder link name in the upload overlay is now missing (link is working though)
    * [x] ~~Sharing a space itself via link, then go to "Shared via link page" -> The name of the space it missing, the folder has a parent folder link instead (I think this is broken on master as well. In general, this is something for later I suppose as this is kind of an edge case)~~ Duplicate of https://github.com/owncloud/web/issues/7103
* The breadcrumb actions on a share itself have some issues:
    * [x] "Details"-action does not set the current share as selected/highlighted file -> sidebar is empty
    * [ ] "Rename"-action doesn't even open the modal window 
    * [x] "Delete"-action declines the share -> better naming? Fixed via https://github.com/owncloud/web/pull/7379
![image](https://user-images.githubusercontent.com/50302941/166899435-5bdf3152-405b-46bc-a9cd-43d0709bcd63.png)

**Comment by JanAckermann** [comment]
Note: Rename doesn't appear anymore. We need to enable rename again and check the integration with the backend

**Comment by JanAckermann** [comment]
@kulmann 
>Uploading a file to a space root -> the parent folder link name in the upload overlay is now missing (link is working though)

![image](https://user-images.githubusercontent.com/26169327/182125829-63a5699d-0f85-4847-bae2-fb82ca75af5f.png)
Is this solved?

**Comment by kulmann** [comment]
> @kulmann
> 
> > Uploading a file to a space root -> the parent folder link name in the upload overlay is now missing (link is working though)
>  Is this solved?

Looks like it 👍

**Comment by JanAckermann** [comment]
Renaming a received share is blocked by: https://github.com/owncloud/ocis/issues/4322
